ocf_data_sampler/numpy_sample/satellite.py

```diff
@@ -0,0 +1,30 @@
+"""Convert Satellite to NumpySample"""
+import xarray as xr
+
+
+class SatelliteSampleKey:
+
+    satellite_actual = 'satellite_actual'
+    time_utc = 'satellite_time_utc'
+    x_geostationary = 'satellite_x_geostationary'
+    y_geostationary = 'satellite_y_geostationary'
+    t0_idx = 'satellite_t0_idx'
+
+
+def convert_satellite_to_numpy_sample(da: xr.DataArray, t0_idx: int | None = None) -> dict:
```

Comment:
Likewise. Not strictly necessary - but perhaps validation check for required input variables? Safe measure